go-controller/pkg/util/go_ovn.go

```diff
@@ -0,0 +1,543 @@
+package util
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"fmt"
+	"io/ioutil"
+	"syscall"
+
+	goovn "github.com/ebay/go-ovn"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/config"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util/ovnbindings"
+	"k8s.io/klog"
+)
+
+type OVNClient struct {
+	client goovn.Client
+}
+
+var _ ovnbindings.OVNInterface = &OVNClient{}
+
+func NewOVNNBClient() (ovnbindings.OVNInterface, error) {
```

Comment:
Is there going to be ever a need for more than one instance of ovnNBClient needed? in other words, would a singleton suffice?

Reply:
You could if we ever decide to have more than one controller and sub-divide processing.